FlatCache/FlatCache.swift

```diff
@@ -32,7 +32,7 @@ private extension Cachable {
     static var typeName: String {
         return String(describing: self)
     }
-
+    
```

Comment:
Can you strip these whitespace changes?

Reply:
Yeah I noticed somehow the Xcode auto refactor decided to throw in some tabs 😐